code/modules/projectiles/guns/magic/staff.dm

```diff
@@ -1,6 +1,30 @@
 /obj/item/weapon/gun/magic/staff/
 	slot_flags = SLOT_BACK
 
+/obj/item/weapon/gun/magic/staff/death
+	name = "staff of death"
+	desc = "An artefact that spits deadly bolts of energy which overwhelm a victim's body with pure energy, slaying them without fail"
+	fire_sound = "sound/magic/WandoDeath.ogg"
+	ammo_type = /obj/item/ammo_casing/magic/death
+	icon_state = "scythe0"
```

Comment:
I'm not certain, but the scythe item only gives icon_state = "scythe0" instead of a full path, and I can't find the base /obj/item/weapon anywhere to see what the full path to the icon file is, so unfortunately I can't use the full path instead.

Although the base item /obj/item/weapon/gun/magic doesn't give a full path, nor does /obj/item/weapon/gun, so it's probably ok.